posthog/queries/trends.py

```diff
@@ -246,6 +246,28 @@ def get_aggregate_total(query: QuerySet, entity: Entity) -> int:
     return entity_total
 
 
+def aggregate_cohort_filter(
+    filtered_events: QuerySet, filter: Filter, entity: Entity, team_id: int, breakdown_value: Union[str, int]
+) -> int:
+    if len(filtered_events) == 0:
+        return 0
+
+    filtered_events = filtered_events.filter(
```

Comment:
Q: Why is this filter call separate from the one below? Would it make sense to collapse them, e.g.

```
    breakdown_filter:Dict[str, Union[bool, str, int] = add_cohort_annotations(...)
    if filter.breakdown_type == "cohort":
        breakdown_filter = {"cohort_{}".format(breakdown_value): True}
    else:
        breakdown_filter = {"properties__{}".format(filter.breakdown): breakdown_value}
```

Reply:
well what i just realized from your comment is that we don't even need to add_cohort_annotations again they're already there!